tests/Microsoft.Identity.Test.Integration.net45/SeleniumTests/InteractiveFlowTests.cs

```diff
@@ -190,6 +205,8 @@ private async Task<AuthenticationResult> RunTestForUserAsync(LabResponse labResp
             userCacheAccess.AssertAccessCounts(1, 1); // the assert calls GetAccounts
             Assert.IsFalse(userCacheAccess.LastNotificationArgs.IsApplicationCache);
 
+            AssertTelemetryHeaders(factory, LastRequestExpectedTelemetryBasic);

```

Comment:
So this is a bit concerning. It seems that even though the test is driving MSAL thorugh a lot, the telemetry values stay the same?